doc/dftb+/manual/dftbp.tex

```diff
@@ -1186,9 +1185,16 @@ \section{Hamiltonian}
     }
   }
 \end{verbatim}
-  Net charge on atoms not appearing in any \is{AtomCharge}
-  specification is set to be zero.
+Net charges on atoms not appearing in any \is{AtomCharge}
+specification are set to be zero.
 
+Finally, the charges on all atoms can be set using the \kw{Random} keyword:
+\begin{verbatim}
+  InitialCharges = { # Specifying charge for various species
+      Random = Yes
+  }
+\end{verbatim}
+These will then be supplied by the code's internal random number generator.
 
```

Comment:
I think, that does not fit to the logic. For me
```
InitialCharges {
  RandomCharges {}
}
```
would be more natural. Also giving us the chance to parametrize it further (e.g. specify the range of the atoms, the range of the randomness).